docs/design/autoware-interfaces/ad-api/list/api/perception/object_recognition.md

```diff
@@ -0,0 +1,24 @@
+---
+title: /api/perception/object_recognition/objects
```

Comment:
I don't think API needs classification like `object_recognition` and it can be a simpler name such as:
- `/api/perception/objects`
- `/api/perception/dynamic_objects`

@mitsudome-r @yukkysaito What do you think about this? How should objects be classified for perception?

Reply:
I agree `/api/perception/dynamic_objects` or `/api/perception/objects`